mitodl/hugo-course-publisher
# desktop home page

#### Pre-Flight checklist


#### What are the relevant tickets?
https://github.com/mitodl/hugo-course-publisher/issues/170

#### What's this PR do?
Adds a home page to the project, based o https://projects.invisionapp.com/d/main?origin=v7#/console/18609637/426576569/inspect?scrollOffset=0

#### How should this be manually tested?
Run the site and visit the root (http://localhost:3000) or visit the preview deploy.  Verify that the page looks correct in desktop web browsers.

#### Screenshots (if appropriate)
![Screen Shot 2020-08-05 at 1 54 21 PM](https://user-images.githubusercontent.com/12089658/89447053-606ffa80-d723-11ea-8b3c-5048fc855480.png)
